pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -180,8 +180,12 @@ let
       INET_UDP_DIAG     = module;
       INET_RAW_DIAG     = whenAtLeast "4.14" module;
       INET_DIAG_DESTROY = whenAtLeast "4.9" yes;
+    } // optionalAttrs (stdenv.hostPlatform.system == "aarch64-linux") {
+      # Needed for policy routing. Present in x86 defaults, absent from the aarch64 ones
+      CONFIG_IP_MULTIPLE_TABLES = yes;
     };
 
+
```

Comment:
? :)

Reply:
Yet another instance of clumsy thumbs :D